modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -802,6 +830,7 @@ void ONNXImporter::parseReduce(LayerParams& layerParams, const opencv_onnx::Node
 {
     opencv_onnx::NodeProto node_proto = node_proto_;
     const std::string& layer_type = node_proto.op_type();
+    const std::string output_name = node_proto.output(0);
```

Comment:
Reference?

`const std::string&`

Reply:
`node_proto`'s output name is modified below.